Fake docs improvement (#2360)
sinonjs/sinon: docs/release-source/release/fakes.md

### Introduction

> `fake` is available in Sinon from v5 onwards. It allows creation of a `fake` `Function` with the ability to set a default [behavior](#fakes-with-behavior). Set the [behavior](#fakes-with-behavior) using `Functions` with the same API as those in a [`sinon.stub`][stubs].

Reviewer: ```suggestion
`fake` is available in Sinon from v5 onwards. It allows creation of a `fake` `Function` with the ability to set a default [behavior](#fakes-with-behavior). The available [behaviors](#fakes-with-behavior) for the most part match the API of a [`sinon.stub`][stubs].
```

### Instance properties

> The instance properties are the same as a [`sinon.spy`][spies]. The following properties are just a few of them, you can refer to

Reviewer: ```suggestion
The instance properties are the same as those of a [`sinon.spy`][spies]. The following examples showcase just a few of the properties available to you. Refer to the [spy docs][spies] for a complete list.
```
